changelog.d/gh-5717.fixed

```diff
@@ -0,0 +1 @@
+Rust: Fixed a bug with matching for scoped identifiers
```

Comment:
Same here, if you just summarized the what/why of your PR comment here it would make a great changelog entry. Then Bruno and the technical writers take these changelog entries and they produce some release notes, so they appreciate the extra details.